app/views/layouts/application.html.erb

```diff
@@ -89,7 +89,7 @@
                                     <%= link_to 'Register', "/sign_up" %>
                                 </div>
 
-                            <% if current_user.role == "local_school_admin" %>
+                            <% if current_user && current_user.role == "local_school_admin" %>
```

Comment:
This should only show if there is no `current_user` so check for `user_signed_in?`